strawberry/subscriptions/protocols/graphql_transport_ws/handlers.py

```diff
@@ -277,12 +282,12 @@ async def cleanup_operation(self, operation_id: str) -> None:
             return
         generator = self.subscriptions.pop(operation_id)
```

Comment:
```suggestion
        result_source = self.subscriptions.pop(operation_id)
```

Reply:
Renamed generator in commit: https://github.com/strawberry-graphql/strawberry/pull/2141/commits/0f70ef209bc178def787118d7088ec1b5165a542